bindings/matrix-sdk-ffi/src/platform.rs

```diff
@@ -611,6 +643,33 @@ fn setup_lightweight_tokio_runtime() {
     }));
 }
 
+#[matrix_sdk_ffi_macros::export]
+pub fn reload_tracing_file_writer(
```

Comment:
I was split whether this should have a more generic name such as `update_platform(file_configuration:)` that would align closer with `init_platform`, but went with this for now. Happy to update if anyone has a preference :)

Also realising this needs a doc to say you must call `init_platform` first (especially if not renamed).